src/object/srv_obj_migrate.c

```diff
@@ -2539,6 +2539,13 @@ migrate_cont_iter_cb(daos_handle_t ih, d_iov_t *key_iov,
 			break;
 	}
 
+	/* FIXME: keep container handle open until all ULTs completes. */
+	for (rc1 = 0; rc1 < 6; rc1++) {
+		if (tls->mpt_obj_generated_ult == tls->mpt_obj_executed_ult)
+			break;
+		sleep(1);
```

Comment:
Is 6 second enough? or we need to use "do{}while()"? On the other hand, one second for ULT schedule may be too coarse-grained, dss_sleep() is more general used for such purpose in DAOS.

Reply:
sleep(1) won't work here, it'll put the whole xstream(thread) into sleep for 1 second.

If the purpose is to wait for all migrating ULTs exit, ABT_cond could be used here. Should we fix this in a separate patch? @wangdi1 , any thoughts?

Reply:
Re-implemented this more correctly. Now it check the status of rebuild progress from all xstreams.

Reply:
Hmm, so close container before migrating object might cause issue?  Agree with niu and Nasf, this is too hacky

Also migrate_cont_iter_cb() happens in xstream 0, and obj_generated_ult is increased in xstream 0, but obj_exected_ult will be increased in other ULTs, see migrate_obj_ult().  
.